app/src/main/java/net/tensory/rxjavatalk/house/HouseFragment.java

```diff
@@ -27,6 +27,7 @@
 public class HouseFragment extends Fragment {
 
     public static final String ARG_HOUSE = "ARG_HOUSE";
+    private static final long ANIMATION_DURATION = 2000L;
```

Comment:
This one I made longer, because it was going too fast to use for demonstration purposes, I wanted to give people a chance to see the update in the Activity Feed and then look at the dashboard ot see that the values updated correctly